% Chapter 1

\chapter{Introduction} % Main chapter title

\label{Chapter1} % For referencing the chapter elsewhere, use \ref{Chapter1} 

% \lhead{Chapter 1. \emph{Introduction}} % This is for the header on each page - perhaps a shortened title

%----------------------------------------------------------------------------------------
\section{Morphology}

Parvoviruses belong to the smallest of isometric viruses. A linear single-stranded DNA genome of about 5 kb is packaged into the virus capsid \cite{pmid4975639, pmid5264145, pmid5429749}. They are non-enveloped and their diameters range from 215 \r{A} (Penaeus stylirostris densovirus, PstDNV) to 255 \r{A} (CPV). The icosahedral nature of parvoviruses was shown unambiguously by X-ray crystallography. The capsid surface of some, particularly invertebrate, parvoviruses appears to be smooth (Galleria mellonella densovirus, GmDNV) whereas others (Adeno-associated virus-2, AAV-2) are spiky at the 3- or 5-fold symmetry axes \cite{pmid10497831, icvt}.      


\nomenclature{PstDNV}{Penaeus stylirostris densovirus}
\nomenclature{GmDNV}{Galleria mellonella densovirus}
\nomenclature{DNA}{Deoxyribonucleic acid}




\section{Physicochemical properties}
Infectious parvovirus virions are composed of about 75 \% protein and 25 \% DNA. Their molecular weight is approximately 5.5-6.2 x 10\textsuperscript{6}. The virion buoyant density is 1.39 to 1.43 gcm\textsuperscript{-3}, measured in CsCl gradients \cite{CsCl, pmid4317344}. Since parvoviruses are devoid of a lipid envelope, mature virions are stable in the presence of lipid solvents. In particular, animal parvoviruses show considerable heat resistance. Most species remain stable and infectious on exposure to pH 7-9 or incubation at 56 \textcelsius~ for 60 min \cite{pmid12935806, pmid12385412, pmid17880601, pmid19039515}. Only harsh conditions, such as treatment with formalin, $\beta$-propiolacetone, hydroxylamine, ultraviolet light, and oxidizing agents as for example sodium hypochlorite ensure effective virus inactivation \cite{pmid4213983, pmid3416941, pmid7848502, pmid1520981}.    

\section{Taxonomy}
The classification of the \textit{Parvoviridae} family is based on morpholological and functional characteristics. Parvoviruses are common animal and insect pathogens that belong to the smallest DNA-containing viruses. Hence the prefix "parvum" which means small in Latin. The name "parvovirus" was first introduced to the literature by Carlos Brailovsky, in an early attempt to establish a latinized binomial taxonomy system for viruses, in 1966 \cite{pmid5902774}.   
The \textit{Parvoviridae} family comprises all non-enveloped, isometric, small DNA viruses that contain linear single-stranded genomes. Indeed, parvoviruses are the only viruses in the known biosphere that have both single-stranded and linear DNA genomes. The encapsidated single genomic molecule is 4-6 kb in length and terminates in palindromic duplex hairpin telomers. As a consequence of such a simple genome, parvoviruses are highly dependent on their host for diverse functions in their reproduction \cite{pmid10497831, parvoviruses}. The terminal hairpins are fundamental for the unique replication strategy of the \textit{Parvoviridae} family and hence serve as an invariant hallmark for classification.
Members of the family \textit{Parvoviridae} infect a wide variety of hosts, ranging from insects to primates.
Depending on this feature, the \textit{Parvoviridae} are subdivided into \textit{Parvovirinae} infecting vertebrates and \textit{Densovirinae} infecting insects and other arthropods, respectively. The \textit{Parvovirinae} subfamily is further subdivided into eight genera: \textit{Amdoparvovirus}, \textit{Aveparvovirus}, \textit{Bocaparvovirus}, \textit{Copiparvovirus}, \textit{Dependoparvovirus}, \textit{Erythroparvovirus}, \textit{Protoparvovirus} and \textit{Tetraparvovirus}. The subdivision into the eight genera is based on differences in transcription maps, organization of the ITRs, the ability to replicate efficiently either autonomously or with helper virus, the sense of the ssDNA that is packaged in separate virions, and sequence homology amongst the \textit{Parvovirinae} subfamily \cite{pmid11222696, icvt}.

\subsection{The \textit{Parvovirinae} subfamily}
   


\subsubsection{\textit{Amdoparvovirus}}
The genus \textit{Amdoparvovirus} shares most characteristics with the genera \textit{Bocaparvovirus} and \textit{Protoarvovirus}. Mature virions exclusively contain negative strand genomic DNA of approximately 4.8 kb in length harbouring dissimilar palindromic sequences at each end \cite{pmid6252342, pmid2843669}. Only two distant species have been reported. Firstly, \textit{Carnivore amdoparvovirus 1}, which comprises only Aleutian mink disease virus (AMDV) and secondly, \textit{Carnivore amdoparvovirus 2}, which encompasses solely gray fox amdovirus (GFAV) \cite{pmid22000359}. 
Permissive replication is tightly restricted to Crandell feline kidney cells. In contrast to the members of the genera \textit{Bocaparvovirus} and \textit{Protoparvovirus}, the virion surface displays three mounds elevated around the threefold icosahedral axis of symmetry. However, several structure features were ascertained to be similar to those found in B19V, CPV, FPV, and MVM. Such appearance is comparable to those observed for the genus \textit{Dependoparvovirus} \cite{pmid10400786}. Remarkably, there is no evidence of a phospholipase 2A enzymatic core within the naturally truncated N-VP1 terminus of members belonging to the genus \textit{Amdoparvovirus} as it is common to the other genera of the subfamily \textit{Parvovirinae} \cite{icvt}. 

Transcription profile of AMDV (QIU, Pintel, 2006)      

\nomenclature{AMDV}{Aleutian mink disease virus}
\nomenclature{GFAV}{Gray fox amdovirus}
\nomenclature{PLA2}{Phospholipase A2}

\subsubsection{\textit{Aveparvovirus}}

\textit{Aveparvovirus} is a new genus within the \textit{Parvovirinae} subfamily that comprises the species chicken parvovirus and turkey parvovirus. The name \textit{Aveparvovirus} is derived from \underline{av}ian parvoviruses, referring to the hosts from which the members were isolated. Although those viruses have been identified for years in the intestinal tracts of poultry \cite{pmid18766849, pmid6847550, pmid2995561}, analysis of the complete nucleotide sequence has been reported only recently. Phylogenetic study of the genomic sequences revealed that interestingly, ChPV and TuPV do not group phylogenetically with GPV and DPV, that are members of the genus \textit{Dependoparvovirus}. It was clearly demonstrated that ChPV, along with the closely related TuPV, represents the prototype of a novel genus within the \textit{Parvovirinae} subfamily \cite{pmid25510852, pmid18622862}. Identical direct repeat sequences flank the genome at both the 3' and the 5' end. Each of which contains a 39 nt ITR that is predicted to form a hairpin structure. ChPV and TuPV feature an overall genome organization similar to that of members of the genus \textit{Bocaparvovirus} \cite{pmid20097398}. 
Although it has been demonstrated that ChPV can induce clinical signs in broiler chickens that show characteristics of the runting-stunting syndrome (RSS) \cite{pmid18766892}, the role of avian parvoviruses in the aetiology of enteric diseases in poultry still remains to be demonstrated. RSS, also referred to as malabsorption syndrome, is characterized by significantly decreased egg hatchability, poorly developed hatched chickens, serious growth retardation, diarrhoea, enteritis, disturbed feathering, low vitality, and bone disorders \cite{pmid7150147, pmid6281962, pmid8363509}. Currently, the pathogenicity of TuPV has not been investigated yet. The predominant enteric diseases in turkeys are known as poult enteritis complex (PEC) \cite{pmid10935280} or the more drastic poult enteritis mortality syndrome (PEMS) \cite{PEMS}. Understanding the role of avian parvoviruses in PEMS and RSS is of great interest since the economic impact from enteric disease is primarily due to poor production, due to poor feed conversion efficiency, as well as due to the increase of costs for therapy \cite{pmid18622862}. 
  
    



\nomenclature{ChPV}{Chicken parvovirus}
\nomenclature{TuPV}{Turkey parvovirus} 
\nomenclature{GPV}{Goose parvovirus}
\nomenclature{DPV}{Duck parvovirus}
\nomenclature{RSS}{Runting-stunting syndrome}
\nomenclature{PEC}{Poult enteritis complex}
\nomenclature{PEMS}{Poult enteritis mortality syndrome}

\subsubsection{\textit{Copiparvovirus}}

\subsubsection{\textit{Bocaparvovirus}}
The name of the genus is derived from \underline{bo}vine and \underline{ca}nine, referring to the two hosts of the first identified members of this genus.  The genomes of members of the genus \textit{Bocaparvovirus} are quite distinct from all other viruses in the subfamily \textit{Parvovirinae}. As the members of the genera \textit{Protoparvovirus} and \textit{Amdoparvovirus} they contain non-identical imperfect palindromic sequences at both ends of their 5.5 kb genome. Mature virions contain mainly, but not exclusively, negative strand ssDNA \cite{pmid3783814,pmid12441065}.
All RNA transcripts are generated from a single P4 promoter at the left-hand end of the genome. BPV RNA transcripts are alternatively spliced and polyadenylated either at an internal site or at the 3’-end of the genome \cite{pmid17715221}. Noteworthy, bovine parvovirus (BPV), the main representative, encodes a 22.5 kDa nuclear phosphoprotein, NP-1, whose function still remains unknown. This protein is distinct from any other parvovirus-encoded polypeptide \cite{pmid6319731}.
A human bocavirus was first described in 2005, when it was detected in nasopharyngeal aspirates of young children with respiratory tract infection \cite{pmid11562506, pmid16118271}. More recently, HBoV has been identified in diarrheal feces of children with gastroenteritis \cite{pmid17553287}. HBoV infection is associated with acute respiratory symptoms and is usually detected in children under 2 years of age \cite{pmid17122013, pmid16517912, pmid17041855}. HBoV infections have been reported world-wide and HBoV was often isolated in respiratory samples of diseased as well as asymptomatic patients sometimes long after the primary infection. Therefore, it can be frequently detected even though it is not likely acting as a pathogen, thus complicating the use of PCR in diagnostics. Furthermore, long-term persistence may explain that HBoV infection among adults was predominantly reported in association with immunosuppression or immunodeficiency \cite{pmid17041855, pmid17176591}.         



\nomenclature{BPV}{Bovine parvovirus}
\nomenclature{kDa}{Kilodalton}
\nomenclature{Da}{Dalton}

\subsubsection{\textit{Dependoparvovirus}}
Positive and negative strand ssDNA is distributed indifferently among mature virions belonging to the genus \textit{Dependoparvovirus} \cite{pmid5014934, pmid5264145}. The 4.7 kb DNA molecule contains identical ITRs of 145 nt, the first 125 nt of which form a palindromic sequence \cite{pmid6246271}. Three mRNA promoters that are located at map units 5, 19, and 40 initiate transcription that can be terminated in two polyadenylation sites located at the right-hand end or alternatively, in the middle of the genome \cite{pmid6253077, pmid6281463}. Common for all currently accepted replication-defective members of the genus \textit{Dependoparvovirus} is their strict dependence upon helper adenoviruses or herpesviruses \cite{pmid4318977, pmid6270377, pmid5227666}. Therefore, their host range tropism strongly depends on the one of the helper virus. 
The only exceptions are the autonomously replicating duck and goose parvoviruses which are also comprised within the \textit{Dependoparvovirus} genus based on phylogenetic analysis \cite{icvt}. The most important members of this genus are the adeno-associated viruses (AAV). They attracted considerable interests since at least one of them, AAV-2, has been reported to integrate site-specifically into human chromosome 19 \cite{pmid2156265, pmid1653762, pmid1334463, pmid1657596}. This characteristic makes AAV a promising candidate for creating viral vectors for gene therapy \cite{pmid18854481, pmid21499295}. As a well characterized member of the \textit{Dependoparvoviruses} AAV-2 represents the model virus among this genus.  


\subsubsection{\textit{Erythroparvovirus}}
Equivalent numbers of positive and negative sense ssDNA are packaged into infectious virions of the genus \textit{Erythroparvovirus}. As in the case with the genus \textit{Dependoparvovirus}, the 5.5 kb ssDNA molecule contains identical ITRs of 383 nt in length at both the 3’ and the 5’ end. The first 365 nt of those secondary elements form palindromic sequences \cite{pmid2408228}. Transcription is regulated by a single mRNA promoter located at map unit 6 \cite{pmid3824910}. A distal polyadenylation site for use in termination of RNA synthesis is located at the far right side. Additionally, transcripts may be terminated at an unusual internal polyadanylation site in the middle of the genome \cite{pmid3599180}. Viruses belonging to this genus are highly erythrotropic, meaning that efficient replication only occurs in rapidly dividing erythroid progenitor cells (EPCs) such as erythroblasts and megakaryocytes present in the bone marrow.
B19V, a widespread human pathogen that causes fifth disease, polyarthropathia, anemic crises in children with underlying hematological diseases (e.g. sickle cell anemia or thalassemia) and intrauterine infections (with hydrops fetalis in some cases) \cite{pmid12097253} represents the model virus among the genus \textit{Erythroparvovirus}. 

\nomenclature{Nt}{Nucleotide}
\nomenclature{ITR}{Inverted terminal repeat}
\nomenclature{Kb}{Kilo base}
\nomenclature{EPC}{Erythroid progenitor cell}


\subsubsection{\textit{Protoparvovirus}}
Kilham Rat virus (KRV), a member of the genus \textit{Protoparvoviruses} was the first member of the subfamily \textit{Parvovirinae} to be discovered in 1959 \cite{pmid13669314}. Some members of the genus contain positive strand DNA in variable proportions up to 50 \% \cite{pmid6694260}. However, in mature virions of most members, virtually only negative strand DNA occurs. What they have in common are their hairpin structures at both the 5’ and 3’ ends of the linear 5 kb ssDNA molecule that differ in both sequence and predicted structure \cite{pmid6298737}. Transcription of the genome is regulated by two mRNA promoters at map units 4 and 39 \cite{pmid6828378}. There is only one polyadenylation site at the 3’ end. 
Viral replication provokes characteristic cytopathic effects in cell culture. Many species display hemagglutination with erythrocytes of one or several species, but not enforcedly of their natural host \cite{pmid5083410}. The genus \textit{Protoparvovirus} is primarily represented by MVM \cite{icvt, parvoviruses}.       

\nomenclature{KRV}{Kilham rat virus}

\subsubsection{\textit{Tetraparvovirus}}
The genus \textit{Tetraparvovirus} is a new genus that arose recently. To date, six species have been discovered, which were isolated from humans \cite{pmid15956568}, chimpanzees, baboons \cite{pmid20668071}, cows, pigs \cite{pmid18632954, pmid20653980, pmid22247522}, as well as sheep \cite{pmid21980506}. RNA transcripts that encode the NS-proteins or the VP-proteins are generated from two promoters that are located at map units 6 and 38, respectively. Transcription can be terminated in two polyadenylation sites located at the right-hand end of the genome or alternatively, at an internal polyadenylation site. Since the full-length genome has not been sequenced yet, information of the terminal repeats is still lacking \cite{pmid22044541}. Analysis of the NS1 protein revealed a G2/M cell cycle arrest induced in NS1-expressing hematopoietic stem cells that clearly involved the predicted helicase motifs \cite{pmid8106366, pmid9261429, pmid7966641} of NS1. To date, no PLA2-like activity of expressed VP1u polypeptides has been demonstrated for any member of the genus \textit{Tetraparvovirus} \cite{pmid22044541}. PARV4 is one of the only four groups of parvoviruses that is known to infect humans besides B19V, HBoV, and AAV. It was first reported in an intravenous drug user who was positive for HBV infection in 2005. The patient suffered from arthralgia, confusion, diarrhea, fatigue, neck stiffness, night sweat, pharyngitis, and vomiting. PARV4 represents a phylogenetic deeply rooted lineage between avian dependoviruses and bovine parvovirus type 3 \cite{pmid15956568}. So far, most evidence about PARV4 transmission comes from patients who had engaged in high risk behaviour for blood borne viral infections, where PARV4 infection basically was observed to be strongly associated with HCV and HIV infection \cite{pmid22492853, pmid22235298, pmid17397006}. However, there are several reports of parenteral transmission in the absence of HIV, HBC, or HCV. PARV4 IgG has been documented independently from other blood borne viruses among injecting drug users \cite{pmid23283958}, in haemophilia patients \cite{pmid22043925}, and in patients who were subjected to intra-muscular injections in the past \cite{pmid22469425}. Currently, no definitive clinical syndrome was associated with PARV4 infection and there is no evidence for a potential pathogenicity of related hokoviruses in animals \cite{pmid18632954}. PARV4 viraemia appears to be asymptomatic \cite{pmid20587191} and co-existing blood borne viruses do not increase severity \cite{pmid22235298}.   

  








 


\nomenclature{PARV4}{Parvovirus 4}
\nomenclature{HBoV}{Human Bocavirus}
\nomenclature{HIV}{Human immunodeficiency virus}
\nomenclature{HCV}{Hepatitis C virus}
\nomenclature{HBV}{Hepatitis B virus}
\nomenclature{ORF}{Open reading frame}
\nomenclature{VP1u}{VP1 unique region}

\clearpage



\begin{tiny}
\begin{center}

\begin{longtable}{p{0.7in} p{1.65in} p{1.6in} p{0.55in} p{0.5in}}

\caption[Taxonomy for the subfamily \textit{Parvovirinae}]{The type species for each genus is indicated in bold type. \cite{pmid24212889}} \\

\textbf{Genus} & \textbf{Species} & \textbf{Virus or virus variants} & \textbf{Abbr.} & \textbf{ACNO}\footnotemark\\
\hline
\endhead % all the lines above this will be repeated on every page
\textit{Amdoparvovirus} & \textbf{\textit{Carnivore amdoparvovirus 1}} & Aleutian mink disease virus & AMDV & JN040434 \\
 & \textit{Carnivore amdoparvovirus 2} & Gray fox amdovirus & GFAV & JN202450 \\
\textit{Aveparvovirus} & \textit{\textbf{Galliform aveparvovirus 1}} & Chicken parvovirus & ChPV & GU214704 \\
& & Turkey parvovirus & TuPV & GU214706 \\
\textit{Bocaparvovirus} & \textit{Carnivore bocaparvovirus 1} & Canine minute virus & CnMV & FJ214110 \\
& \textit{Carnivore bocaparvovirus 2} & Canine bocavirus 1 & CBoV & JN648103 \\
& \textit{Carnivore bocaparvovirus 3} & Feline bocavirus & FBoV & JQ692585 \\
& \textit{Pinniped bocaparvovirus 1} & California sea lion bocavirus 1 & CslBoV1 & JN420361\\
& & California sea lion bocavirus 2 & CslBoV2 & JN420366 \\
& \textit{Pinniped bocaparvovirus 2} & California sea lion bocavirus 3 & CslBoV3 & JN420365 \\
 & \textit{Primate bocaparvovirus 1} & Human bocavirus 1 & HBoV1 & JQ923422 \\
 & & Human bocavirus 3 & HBoV3 & EU918736 \\
 & & Gorilla bocavirus & GBoV & HM145750 \\
 & \textit{Primate bocaparvovirus 2} & Human bocavirus 2a & HBoV2a & FJ973558 \\
 & & Human bocavirus 2b & HBoV2b & FJ973560 \\
 & & Human bocavirus 2c & HBoV2c & FJ170278 \\
 & & Human bocavirus 4 & HBoV4 & FJ973561 \\
 & \textit{\textbf{Ungulate bocaparvovirus 1}} & Bovine parvovirus & BPV & DQ335247 \\
 & \textit{Ungulate bocaparvovirus 2} & Porcine bocavirus 1 & PBoV1 & HM053693 \\ 
 & & Porcine bocavirus 2 & PBoV2 & HM053694 \\
 & & Porcine bocavirus 6 & PBoV6 & HQ291309 \\
 & \textit{Ungulate bocaparvovirus 3} & Porcine bocavirus 5 & PBoV5 & HQ223038 \\
 & \textit{Ungulate bocaparvovirus 4} & Porcine bocavirus 7 & PBoV7 & HQ291308 \\ 
 & \textit{Ungulate bocaparvovirus 5} & Porcine bocavirus 3 & PBoV3 & JF429834 \\ 
 & & Porcine bocavirus 4-1 & PBoV4-1 & JF429835 \\
 & & Porcine bocavirus 4-2 & PBoV4-2 & JF429836 \\
 \textit{Copiparvovirus} & \textit{\textbf{Ungulate copiparvovirus 1}} & Bovine parvovirus 2 & BPV2 & AF406966 \\
 & \textit{Ungulate copiparvovirus 2} & Porcine parvovirus 4 & PPV4 & GQ387499 \\
 \textit{Dependoparvovirus} & \textit{\textbf{Adeno-associated dependoparvovirus A}} & Adeno-associated virus-1 & AAV1 & AF063497 \\ 
 & & Adeno-associated virus-2 & AAV2 & AF043303 \\
 & & Adeno-associated virus-3 & AAV3 & AF028705 \\
 & & Adeno-associated virus-4 & AAV4 & U89790 \\
 & & Adeno-associated virus-6 & AAV6 & AF028704 \\
 & & Adeno-associated virus-7 & AAV7 & AF513851 \\
 & & Adeno-associated virus-8 & AAV8 & AF513852 \\
 & & Adeno-associated virus-9 & AAV9 & AX753250 \\
 & & Adeno-associated virus-10 & AAV10 & AY631965 \\ 
 & & Adeno-associated virus-11 & AAV11 & AY631966 \\ 
 & & Adeno-associated virus-12 & AAV12 & DQ813647 \\
 & & Adeno-associated virus-13 & AAV13 & EU285562 \\
 & & Adeno-associated virus-S17 & AAVS17 & AY695376 \\
 & \textit{Adeno-associated dependovirus B} & Adeno-associated virus-5 & AAV5 & AF085716 \\
 & & Bovine adeno-associated virus & BAAV & AY388617 \\
 & & Caprine adeno-associated virus & CapAAV & DQ335246 \\
 & \textit{Anseriform dependoparvovirus 1} & Duck parvovirus & DPV & U22967 \\
 & & Goose parvovirus-PT & GPV2 & JF926695 \\
 & & Goose parvovirus & GPV & U25749 \\
 & \textit{Avian dependovirus 1} & Avian adeno-associated virus & AAAV & AY186198 \\
 & \textit{Chiropteran dependoparvovirus 1} & Bat adeno-associated virus & BtAAV & GU226971 \\
 & \textit{Pinniped dependoparvovirus 1} & California sea lion adeno-associated virus & CslAAV & JN420372 \\
 & \textit{Squamate dependoparvovirus 1} & Snake adeno-associated virus & SAAV & AY349010 \\
 \textit{Erythroparvovirus} & \textit{\textbf{Primate erythroparvovirus 1}} & Human parvovirus B19-Au & B19V-Au & M13178 \\
 & & Human parvovirus B19-J35 & B19V-J35 & AY386330 \\
 & & Human parvovirus B19-Wi & B19V-Wi & M24682 \\
 & & Human parvovirus B19-A6 & B19V-A6 & AY064475 \\
 & & Human parvovirus B19-Lali & B19V-Lali & AY044266 \\
 & & Human parvovirus B19-V9 & B19V-V9 & AJ249437 \\
 & & Human parvovirus B19-D91 & B19V-D91 & AY083234 \\
 & \textit{Primate erythroparvovirus 2} & Simian parvovirus & SPV & U26342 \\
 & \textit{Primate erythroparvovirus 3} & Rhesus macaque parvovirus & RhMPV & AF221122 \\
 & \textit{Primate erythroparvovirus 4} & Pig-tailed macaque parvovirus & PtMPV & AF221123 \\
 & \textit{Rodent erythroparvovirus 1} & Chipmunk parvovirus & ChpPV & GQ200736 \\
 & \textit{Ungulate erythroparvovirus 1} & Bovine parvovirus 3 & BPV3 & AF406967 \\
 \textit{Protoparvovirus} & \textit{Carnivore protoparvovirus 1} & Feline parvovirus & FPV & EU659111 \\ 
 & & Canine parvovirus & CPV & M19296 \\
 & & Mink enteritis virus & MEV & D00765 \\
 & & Racoon parvovirus & RaPV & JN867610 \\
 & \textit{Primate protoparvovirus 1} & Bufavirus 1a & BuPV1a & JX027296 \\
 & & Bufavirus 1b & BuPV1b & JX027295 \\
 & & Bufavirus 2 & BuPV2 & JX027297 \\
 & \textit{\textbf{Rodent protoparvovirus 1}} & H-1 parvovirus & H1 & X01457 \\
 & & Kilham rat virus & KRV & AF321230 \\
 & & LuIII virus & LuIII & M81888 \\
 & & Minute virus of mice (prototype) & MVMp & J02275 \\
 & & Minute virus of mice (immunosuppressive) & MVMi & M12032 \\
 & & Minute virus of mice (Missouri) & MVMm & DQ196317 \\
 & & Minute virus of mice (Cutter) & MVMc & U34256 \\
 & & Mouse parvovirus 1 & MPV1 & U12469 \\
 & & Mouse parvovirus 2 & MPV2 & DQ196319 \\
 & & Mouse parvovirus 3 & MPV3 & DQ199631 \\
 & & Mouse parvovirus 4 & MPV4 & FJ440683 \\
 & & Mouse parvovirus 5 & MPV5 & FJ441297 \\
 & & Hamster parvovirus & HaPV & U34255 \\ 
 & & Tumor virus X & TVX & In preparation \\
 & & Rat minute virus 1 & RMV1 & AF332882 \\
 & \textit{Rodent protoparvovirus 2} & Rat parvovirus 1 & RPV1 & AF036710 \\
 & \textit{Ungulate protoparvovirus 1} & Porcine parvovirus Kresse & PPV-Kr & U44978 \\
 & & Porcine parvovirus NADL-2 & PPV-NADL2 & L23427 \\
 \textit{Tetraparvovirus} & \textit{Chiropteran tetraparvovirus 1} & Eidolon Helvum (bat) parvovirus & Ba-PARV4 & JQ037753 \\
 & \textit{\textbf{Primate tetraparvovirus 1}} & Human parvovirus 4 G1 & PARV4G1 & AY622943 \\
 & & Human parv4 G2 & PARV4G2 & DQ873391 \\      
 & & Human parv4 G3 & PARV4G3 & EU874248 \\
 & & Chipmanzee parv4 & Ch-PARV4 & HQ113143 \\
 & \textit{Ungulate tetraparvovirus 1} & Bovine hokovirus 1 & B-PARV4-1 & EU200669 \\
 & & Bovine hokovirus 2 & B-PARV4-2 & JF504697 \\
 & \textit{Ungulate tetraparvovirus 2} & Porcine hokovirus & P-PARV4 & EU200677 \\
 & \textit{Ungulate tetraparvovirus 3} & Porcine Cn virus & CnP-PARV4 & GU938300 \\
 & \textit{Ungulate tetraparvovirus 4} & Ovine hokovirus & O-PARV4 & JF504699 \\
 



\label{Taxonomy for the subfamily Parvovirinae}
     
\end{longtable}
\end{center} 
\end{tiny}



\footnotetext{NIH GenBank accession number}
\nomenclature{NIH}{National institutes of health}



\section{Tropism}
Most parvoviruses, such as MVM, \nomenclature{MVM}{Minute virus of mice} CPV \nomenclature{CPV}{Canine parvovirus} and FPV, \nomenclature{FPV}{Feline parvovirus} show specific host ranges and tissue tropisms. The majority of the parvoviruses are members of those tightly controlled viruses. However, some parvoviruses, as for example many of the AAVs, \nomenclature{AAV}{Adeno-associated virus} infect many hosts and a variety of tissues. Understanding controls of these properties for autonomous parvoviruses show great promise for the therapeutic delivery to be controlled or modified in gene therapy applications \cite{pmid12941411}.  

To date, two independent strains of the parvovirus MVM have been described to occur in mice. Both strains display disparate \textit{in vitro} tropism and \textit{in vivo} pathogenicity despite differing by only 14 amino acids in their capsid proteins. Fist, MVMp, \nomenclature{MVMp}{Prototype strain of MVM} the prototype strain, was originally isolated from a contaminated murine adenovirus stock and was shown to replicate efficiently in mouse fibroblasts \cite{pmid5945715}. Secondly, MVMi,\nomenclature{MVMi}{Immunosuppressive strain of MVM} the immunosuppressive strain, was recovered from an infected EL-4 T-cell lymphoma culture \cite{ pmid1244418}. Although MVMi infection may result in pathology of infected mice, it has been shown that the infection more likely interferes with numerous T-cell functions \textit{in vitro}. The infection rather causes problems for the ongoing study the mice are being used for as the immune system will be activated, the activity of T-lymphocytes or B-lymphocytes will be altered and tumor formation may be suppressed \cite{pmid6457871, pmid6264106, pmid11528091}.

	As above-mentioned, it has been shown that the allotropic MVM strains, MVMi and MVMp, differ in their target cell tropism. In order to map the viral sequences responsible for that difference, chimeric viral genomes were constructed \textit{in vitro} from infectious genomic clones of both strains. The differences in the cell tropism between MVMi and MVMp have been mapped to the capsid gene, in particular to the VP2 residues 317 and 321. Both residues are located at the base of the threefold spike of the virion and are involved in controlling the cell tropism of the two distinct MVM strains \cite{pmid3357208, pmid3392768, pmid3257270}.  Interestingly, residue 321 aligns with residue 323 of CPV, that itself is a critical determinant for CPV host range tropism. Further residues (VP2 residues 399, 553 and 558) were identified in MVMi to be able to confer fibrotropism to single site-directed mutants. Those residues cluster around the twofold dimple-like depression \cite{pmid9817841}. 
	
	In general, tissue tropism determining amino acids were found be located on, or near, the viral surface, mainly by raised structural elements around the threefold axes of symmetry. Differences in tropism and pathogenicity have also been mapped to the capsid proteins of Aleutian mink disease parvovirus \cite{pmid8396664}, porcine parvovirus (PPV) \nomenclature{PPV}{Porcine parvovirus} \cite{pmid8642680}, CPV \cite{pmid3176341, pmid1331498}, and FPV \cite{pmid7513918} in a capsid region analogous to that observed for MVM. Although the same structural element of viruses is involved in mediating host and tissue tropisms, each appears to be affecting a different mechanism. Host ranges of CPV and FPV are controlled by receptor binding, whereas the cell tropisms of MVM appear to be due to restrictions of interactions with intracellular factors \cite{pmid9817841, pmid12941411}.


%----------------------------------------------------------------------------------------

\section{Structure}

\subsection{Parvoviruses in general}

Parvovirus capsids are devoid of a lipid envelope and have an average diameter of 18 to 26 nm. The viral capsid is made up of 60 copies of between two and four structural proteins that overlap each other. For each virus there is one major capsid protein present in the capsid structure. Minor proteins form the same core structure, but differ in the sequence length on their amino termini. The capsid proteins display a T=1 icosahedral symmetry and are variously designated VP1-VP4. Thus, the capsid has a 5-3-2 point group symmetry containing 31 rotational symmetry elements that intersect at the center: six fivefolds, ten threefolds, and fifteen twofolds. 
Despite the differences in protein forms and the low homology between some of the viruses, several structural elements on the capsid surface are common to most parvoviruses. These include raised cylindrical channels at the fivefold axes surrounded by depressed, canyon-like regions. Further shared surface characteristics are protrusions at the threefold axes, termed as spikes or peaks, and dimple-like depressions at the icosahedral twofold axes.  
A common feature of parvoviruses is their high resistance to physicochemical treatments. This stability provides an effective protection to the fragile, condensed genome in the extracellular environment ensuring transmission between their hosts. The ssDNA \nomenclature{ssDNA}{Single stranded DNA } genome consists of approximately 5000 bases, packed as either a positive or, more usually, as a negative sense strand. At the 5’ and 3’ ends, the genome harbors palindromic sequences of about 120 to 250 nucleotides, that form secondary hairpin structures which are essential for the initiation of viral genome replication \cite{pmid16242744, pmid25555192, pmid8392729, pmid9817841, pmid11827486, pmid2006420, pmid12941411}. \\      

\nomenclature{dsDNA}{Double stranded DNA}








\subsection{MVM}

Both DNA-containing full and empty particles were crystallized in the monoclinic space group C2. Following data processing and refinement, the resulting electron density map was interpreted with respect to the amino acid sequence of MVMi. The known CPV structure was used as a phasing model since 52 \% of the 578 amino acids in VP2 of MVM are identical to CPV. The polypeptide chain of the major structural protein, VP2, could be traced from residue 39 to residue 587 at the C-terminus \cite{pmid15299974}.     
The common c-terminal part of the structural proteins has an eight-stranded antiparallel $\beta$-barrel topology, frequently found in viral capsid proteins \cite{pmid2673017}. Large loops between the $\beta$-strands of the $\beta$-barrel that form the principal surface features, particularly the threefold spikes, and determine host-range tropism were found to be quite dissimilar in MVM and CPV. 
The first 37 amino acids are not visible in the electron density map. Since the N-VP2 terminal part contains a predominantly poly-glycine conserved sequence, it might be highly flexible. There is density extending along the fivefold channels of the MVMi capsid that was modeled as the glycine-rich N-terminal region \cite{pmid15299494, pmid8969301}. \textit{In vitro}, trypsin digestion of full MVM virions results in a truncated VP3 polypeptide that still contains the glycine-rich sequence. In this way, most VP2 N-termini can be cleaved. These findings suggest that there is a dynamic situation at the fivefold channel. In one model, one in five amino termini are externalized along the fivefold axes and are accessible for cleavage. Newly created, cleaved N-VP3 termini could withdraw into the virion and be replaced at the surface by an uncleaved N-VP2 terminus. \cite{pmid8503170, pmid9817841}.
A substantial amount of internal electron density could be related to 10 DNA nucleotides that were previously found in the analysis of the structure of CPV \cite{pmid7735832, pmid1616694}. For MVM, 19 additional DNA nucleotides were identified in a difference electron-density map with respect to the data of empty particles. Thus, 29 ordered, or partially ordered, nucleotides per icosahedral asymmetric unit imply that approximately 34 \% of the total genome display icosahedral symmetry. This finding, and the conservation of base-binding sites between MVMi and CPV, identifies a DNA-recognition site on the parvoviral capsid interior \cite{pmid9817841}.    










\section{Nucleic Acid}

\subsection{Genome organization}


\subsection{Transcriptome}

\section{Viral proteins}


\subsection{Structural Proteins}

Three capsid proteins (LRV) \cite{pmid4321164}
Three capsid proteins (AAV) \cite{pmid5132697, pmid5172922}



\subsection{Non-structural proteins}
 

%----------------------------------------------------------------------------------------

\section{}


\subsection{}



\subsection{}

