% Chapter 1

\chapter{Introduction} % Main chapter title

\label{Chapter1} % For referencing the chapter elsewhere, use \ref{Chapter1} 

\lhead{Chapter 1. \emph{Introduction}} % This is for the header on each page - perhaps a shortened title

%----------------------------------------------------------------------------------------

\section{Tropism}
Most parvoviruses, such as MVM, CPV and FPV, show specific host ranges and tissue tropisms. The majority of the parvoviruses are members of those tightly controlled viruses. However, some parvoviruses, as for example many of the AAVs, infect many hosts and a variety of tissues. Understanding controls of these properties for autonomous parvoviruses show great promise for the therapeutic delivery to be controlled or modified in gene therapy applications \cite{pmid12941411}.  

To date, two independent strains of the parvovirus MVM have been described to occur in mice. Both strains display disparate \textit{in vitro} tropism and \textit{in vivo} pathogenicity despite differing by only 14 amino acids in their capsid proteins. First, MVMp, the prototype strain, was originally isolated from a contaminated murine adenovirus stock and was shown to replicate efficiently in mouse fibroblasts \cite{pmid5945715}. Secondly, MVMi, the immunosuppressive strain, was recovered from an infected EL-4 T-cell lymphoma culture \cite{ pmid1244418}. Although MVMi infection may result in pathology of infected mice, it has been shown that the infection more likely interferes with numerous T-cell functions \textit{in vitro}. The infection rather causes problems for the ongoing study the mice are being used for as the immune system will be activated, the activity of T-lymphocytes or B-lymphocytes will be altered and tumor formation may be suppressed \cite{pmid6457871, pmid6264106, pmid11528091}.

	As above-mentioned, it has been shown that the allotropic MVM strains, MVMi and MVMp, differ in their target cell tropism. In order to map the viral sequences responsible for that difference, chimeric viral genomes were constructed \textit{in vitro} from infectious genomic clones of both strains. The differences in the cell tropism between MVMi and MVMp have been mapped to the capsid gene, in particular to the VP2 residues 317 and 321. Both residues are located at the base of the threefold spike of the virion and are involved in controlling the cell tropism of the two distinct MVM strains \cite{pmid3357208, pmid3392768, pmid3257270}.  Interestingly, residue 321 aligns with residue 323 of CPV, that itself is a critical determinant for CPV host range tropism. Further residues (VP2 residues 399, 553 and 558) were identified in MVMi to be able to confer fibrotropism to single site-directed mutants. Those residues cluster around the twofold dimple-like depression \cite{pmid9817841}. 
	
	In general, tissue tropism determining amino acids were found be located on, or near, the viral surface, mainly by raised structural elements around the threefold axes of symmetry. Differences in tropism and pathogenicity have also been mapped to the capsid proteins of Aleutian mink disease parvovirus (ADV) \cite{pmid8396664}, porcine parvovirus (PPV) \cite{pmid8642680}, CPV \cite{pmid3176341, pmid1331498}, and FPV \cite{pmid7513918} in a capsid region analogous to that observed for MVM. Although the same structural element of viruses is involved in mediating host and tissue tropisms, each appears to be affecting a different mechanism. Host ranges of CPV and FPV are controlled by receptor binding, whereas the cell tropisms of MVM appear to be due to restrictions of interactions with intracellular factors \cite{pmid9817841, pmid12941411}.


%----------------------------------------------------------------------------------------

\section{Structure}

\subsection{Parvoviruses in general}

Parvovirus capsids are devoid of a lipid envelope and have an average diameter of 18 to 26 nm. The viral capsid is made up of 60 copies of between two and four structural proteins that overlap each other. For each virus there is one major capsid protein present in the capsid structure. Minor proteins form the same core structure, but differ in the sequence length on their amino termini. The capsid proteins display a T=1 icosahedral symmetry and are variously designated VP1-VP4. Thus, the capsid has a 5-3-2 point group symmetry containing 31 rotational symmetry elements that intersect at the center: six fivefolds, ten threefolds, and fifteen twofolds. 
Despite the differences in protein forms and the low homology between some of the viruses, several structural elements on the capsid surface are common to most parvoviruses. These include raised cylindrical channels at the fivefold axes surrounded by depressed, canyon-like regions. Further shared surface characteristics are protrusions at the threefold axes, termed as spikes or peaks, and dimple-like depressions at the icosahedral twofold axes.  
A common feature of parvoviruses is their high resistance to physicochemical treatments. This stability provides an effective protection to the fragile, condensed genome in the extracellular environment ensuring transmission between their hosts. The ssDNA genome consists of approximately 5000 bases, packed as either a positive or, more usually, as a negative sense strand. At the 5’ and 3’ ends, the genome harbors palindromic sequences of about 120 to 250 nucleotides, that form secondary hairpin structures which are essential for the initiation of viral genome replication \cite{pmid16242744, pmid25555192, pmid8392729, pmid9817841, pmid11827486, pmid2006420, pmid12941411}. \\      









\subsection{MVM}

Both DNA-containing full and empty particles were crystallized in the monoclinic space group C2. Following data processing and refinement, the resulting electron density map was interpreted with respect to the amino acid sequence of MVMi. The known CPV structure was used as a phasing model since 52 \% of the 578 amino acids in VP2 of MVM are identical to CPV. The polypeptide chain of the major structural protein, VP2, could be traced from residue 39 to residue 587 at the C-terminus \cite{pmid15299974}.     
The common c-terminal part of the structural proteins has an eight-stranded antiparallel $\beta$-barrel topology, frequently found in viral capsid proteins \cite{pmid2673017}. Large loops between the $\beta$-strands of the $\beta$-barrel that form the principal surface features, particularly the threefold spikes, and determine host-range tropism were found to be quite dissimilar in MVM and CPV. 
The first 37 amino acids are not visible in the electron density map. Since the N-VP2 terminal part contains a predominantly poly-glycine conserved sequence, it might be highly flexible. There is density extending along the fivefold channels of the MVMi capsid that was modeled as the glycine-rich N-terminal region \cite{pmid15299494, pmid8969301}. \textit{In vitro}, trypsin digestion of full MVM virions results in a truncated VP3 polypeptide that still contains the glycine-rich sequence. In this way, most VP2 N-termini can be cleaved. These findings suggest that there is a dynamic situation at the fivefold channel. In one model, one in five amino termini are externalized along the fivefold axes and are accessible for cleavage. Newly created, cleaved N-VP3 termini could withdraw into the virion and be replaced at the surface by an uncleaved N-VP2 terminus. \cite{pmid8503170, pmid9817841}.
A substantial amount of internal electron density could be related to 10 DNA nucleotides that were previously found in the analysis of the structure of CPV \cite{pmid7735832, pmid1616694}. For MVM, 19 additional DNA nucleotides were identified in a difference electron-density map with respect to the data of empty particles. Thus, 29 ordered, or partially ordered, nucleotides per icosahedral asymmetric unit imply that approximately 34 \% of the total genome display icosahedral symmetry. This finding, and the conservation of base-binding sites between MVMi and CPV, identifies a DNA-recognition site on the parvoviral capsid interior \cite{pmid9817841}.    









Despite the differences in protein forms and the low homology between some of the viruses, it is now clear that several prominent structural elements on the capsid surface are common to most parvoviruses. These include raised regions at the fivefold axes of symmetry, which in some viruses might form a pore into the capsid, depressed regions (canyons) surrounding the fivefold axes, one or three protrusions at or surrounding the threefold axes of symmetry (threefold spikes or peaks) and depressed regions (dimples) at the twofold axes of symmetry (Figure 1 and Figure 2) 


\subsection{}





\subsection{}


\subsection{}

\subsection{}
 

%----------------------------------------------------------------------------------------

\section{}


\subsection{}



\subsection{}

