% Chapter 2

\chapter{Methods} % Main chapter title

\label{Chapter2} % For referencing the chapter elsewhere, use \ref{Chapter2} 

% \lhead{Chapter 2. \emph{Methods}} % This is for the header on each page - perhaps a shortened title

%----------------------------------------------------------------------------------------

\section{Cell Cultures}
A9 ouab\textsuperscript{r}l1 cells, a derivative from the original HGPRT\textsuperscript{-} L-cell line A9 represent a clone resistant to 10\textsuperscript{-3} M ouabain after nitrosoguanidine mutagenesis \cite{pmid14213660}.
NB324K cells are a clone of SV40-transformed \nomenclature{SV40}{Simian vacuolating virus 40 or Simian virus 40} human newborn kidney cells \cite{pmid13911591}. The SV40 large T antigen was detected by immunofluorescent \nomenclature{IF}{Immunofluorescence} staining with monoclonal antibodies \cite{pmid6169844}. \nomenclature{mAb}{Monoclonal antibody} However, NB324K cells do not produce infectious SV40 spontaneously.
Both cell lines, A9 mouse fibroblasts and NB324K cells, were routinely propagated under a minimal number of passages in DMEM \nomenclature{DMEM}{Dulbecco modified Eagle's medium}supplemented with 5 \% of heat inactivated fetal bovine serum at 37 \textcelsius~ in 5 \% CO\textsubscript{2} atmosphere. \nomenclature{FCS}{Fetal calf serum} 


\subsection{Freezing and thawing of cells}
Before use the A9 mouse fibroblasts or NB324K cells were thawed at 37 \textcelsius~ and cultured in 5 mL of pre-warmed DMEM supplemented with 5 \% FCS. The medium was replaced every 3 to 4 days. 
In order to freeze the cells for long storage in liquid nitrogen they were passed the day before, to ensure exponential growth. Subsequently, 7.5 \% DMSO was added and the cells were frozen slowly at -70 \textcelsius~ over night before transfer to liquid nitrogen.

%----------------------------------------------------------------------------------------

\section{Virus Stocks}
\label{Virus Stocks}
Stocks of MVM without detectable levels of VP3 were propagated on A9 mouse fibroblast monolayers. As soon as the cytopathic effect became evident, the supernatant \nomenclature{SN}{Supernatant}was collected and pre-cleared from cell debris by low-speed centrifugation. Thereby, intracellular, VP3 rich capsids were discarded. In order to remove low-molecular contaminants, virus containing SN was pelleted through 20 \% sucrose cushion in PBS by ultra-centrifugation. Virus titers were determined by qPCR \nomenclature{qPCR}{Quantitative PCR} \nomenclature{PCR}{Polymerase chain reaction} as DNA-packaged particles per microliter.   

\subsection{Separation of empty and full capsids}

Sucrose purified capsids were prepared as previously described in section \ref{Virus Stocks}, page \pageref{Virus Stocks}. The virus pellet was resuspended in 10 mL PBS. Caesium chloride was added to a density of 1.38 g/mL adjusted by refractometry ($\eta$=1.371) at 4 \textcelsius. The gradient was centrifuged to equilibrium for 24 h at 41000 rpm and 4 \textcelsius~ in a Beckmann SW-41 Ti rotor. Gradients were fractionated and tested for intact capsids by dot blot analysis using B7 mAb. CsCl was depleted from the corresponding fractions by size-exclusion chromatography through PD-10 desalting columns and the capsids were concentrated in Amicon\textsuperscript{\textregistered} centrifugal filter devices when required.          
   
   








\section{Freezing bacteria stocks in glycerol}
Bacteria were frozen in dry ice. A volume of 700 $\mu$L of the bacteria culture that was grown over night in LB-medium was mixed with 300 $\mu$L of 50 \% glycerol in a cryotube. In order to mix well the glycerol the cryotube was vortexed intensively. Following snap-freeze in dry ice the bacteria were stored at -70 \textcelsius.



\section{Anion-exchange chromatography}
A Mono Q HR 5/5 (Pharmacia) column (5 x 50 mm) was used to analyse viral samples. The Mono Q column was connected to the ÄKTAmicro chromatography system (GE Healthcare) that was operated by the UNICORN control software. The Mono Q column was equilibrated with six column volumes (CV) starting buffer (20 mM Tris-HCl, 1 mM EDTA, pH 7.2). Samples (1 mL) containing at least 10\textsuperscript{10} virus particles in 10 mM Tris-HCl, 1 mM EDTA, pH 8 were applied to the Mono Q column trough a 2 mL loop. After eluting the protein, which did not bind to the column in the starting buffer, a linear salt gradient (0-2 M NaCl) in 20 mM Tris-HCl, 1 mM EDTA, pH 7.2, was applied. Fractions of 0.185 mL were collected in 96-well plates. Viral genomes in each fraction were quantified by qPCR.  

Occasionally, the Mono Q column needed to be washed. Increased back-pressure, colour change at the top of the column, decreased sample recoveries, or loss of resolution indicates that the column matrix requires regeneration. In order to circumvent such problems, the column was washed every tenth run. To eluate contaminants that tightly stick to the column the following harsh conditions were applied to the reversed (bottom to top) Mono Q column. 500 $\mu$L 2 M NaCl solution was injected and subsequently, the column was rinsed with water. Then, 500 $\mu$L 2 M NaOH solution was injected and the column was rinsed with water. Finally, 500 $\mu$L 75 \% acetic acid was injected before the column was re-equilibrated with starting buffer.             

All buffers were filtered and degassed before application to the Mono Q column.

\nomenclature{CV}{Column volume}


\section{Quantitative PCR}
Amplification of MVM DNA and real-time detection of PCR products were performed by using BioRad CFX96 technology with SYBR green supermix. PCR was carried out by using the hot-start iTaq\textsuperscript{\texttrademark} DNA polymerase (Bio-Rad Laboratories) following the manufacturer’s guide-lines. Viral DNA was isolated using DNeasy blood and tissue kit. Elution of the purified vDNA was carried out using 100 $\mu$L elution buffer. As templates 2 $\mu$L of the isolated viral DNA were used for the PCR reaction and were added to the following master mix:\\

\begin{table}[h]
\begin{center}
\begin{tabular}{l r r}
\textbf{Component} & \textbf{Amount} & \textbf{Final concentration}\\
\hline
dH\textsubscript{2}O, PCR grade & 6 $\mu$L & -\\
Forward primer, 10 pM & 1 $\mu$L & 0.5 pM\\
Reverse primer, 10 pM & 1 $\mu$L & 0.5 pM\\
2x IQ\textsuperscript{\texttrademark} SYBR\textsuperscript{\textregistered} Green Supermix & 10 $\mu$L & 1x\\
\hline
\textbf{Total volume} & \textbf{18 $\boldsymbol{\mu}$L} & \\
\end{tabular}
\end{center}
\label{Master mix}
\caption[Master mix for quantitative PCR]{Master mix for quantitative PCR. In order to minimize pipetting errors a master mix was prepared. Following preparation the master mix was distributed across the 96 well plates. The master mix contains all the ingredients which are required for the DNA amplification except the initial DNA template that differs among the samples.}
\end{table} 

To ensure accurate quantification, the 96-well plates containing master mix and template DNA were shortly spun and transferred into the BioRad CFX96 unit. The following PCR program was used for quantification of viral DNA:

\begin{table}[h]
\begin{center}
\begin{tabular}{r l r r}
\textbf{Cycles} & \textbf{Step} & \textbf{Temperature} & \textbf{Time}\\
\hline
1x & Initial denaturation & 95 \textcelsius & 300 s\\
40x & Denaturation & 95 \textcelsius & 15 s \\
 & Annealing & 61 \textcelsius & 15 s \\
 & Extension & 72 \textcelsius & 15 s \\
1x & Final denaturation & 95 \textcelsius & 60 s \\
1x & Melting curve & 65 \textcelsius~up to 95 \textcelsius & 0.1 \textcelsius/s \\
\end{tabular} 
\end{center} 
\label{PCR conditions}
\caption[PCR conditions]
   {PCR conditions for the amplification and real-time detection of MVM DNA.}
\end{table}

To provide standards for sample quantification, serially diluted plasmids containing the entire MVM genomic DNA were used.
For cell number variations that may exist between the samples, the number of applied cells per PCR reaction needed to be quantified for normalization as well. For this purpose quantification of cellular $\beta$-actin gene was performed. After normalization, direct comparison of the results is possible. $\beta$-actin quantification was carried out with the same PCR conditions outlined in table \ref{PCR conditions}, \pageref{PCR conditions}.



\section{Immunoprecipitation}
Either \textit{in vitro} treated viruses or viruses from cell extracts were transferred to LoBind tubes that were pre-blocked with PBS containing 1 \% bovine serum albumin (PBSA 1 \%). The volume was adjusted to 200 $\mu$L with PBSA 1 \%. The antibody was added in excess and incubated with the viral capsids for 1 h at 4 \textcelsius~ on a rotary shaker. Subsequently, 20 $\mu$L protein G-agarose beads were added. Following overnight incubation at 4 \textcelsius~ and centrifugation at 2500 rpm for 5 min the supernatant was discarded. The beads were washed 4 times with PBSA 1 \%. To remove the BSA an additional wash step was carried out with PBS. Finally, the beads were frozen at -20 \textcelsius~ until further use or immediately processed. 

\nomenclature{IP}{Immunoprecipitation}

\section{SDS-PAGE and Western blotting}

Immunoprecipitated capsids were dissolved in 20 $\mu$L protein loading buffer (reference) containing 2 \% SDS and 10 \% glycerol. The samples were boiled at 96 \textcelsius~ for 8 min. Viral proteins were separated through a NuPAGE\textsuperscript{\textregistered} 10 \% Bis-Tris Gel (Invitrogen). The XCell Sure Lock\textsuperscript{\texttrademark} Electrophoresis Cell (Invitrogen) was used to separate the proteins. The gel was first run at 30 V for 10 min to stack the proteins. In this way, sharper bands could be achieved. Separation of the different proteins was accomplished at 200 V. Following separation, the proteins were blotted on a methanol activated, porous, 0.2 $\mu$m polyvinylidene fluoride (PVDF) Immobilon\textsuperscript{\textregistered} Transfer Membrane (EMD Millipore). Blotting was carried out at 30 V for 1 h 10 min using XCell II\textsuperscript{\texttrademark} Blot Module (Invitrogen). 
The membrane was blocked in TBS-T buffer (reference) supplemented with 5 \% milk overnight at 4 \textcelsius. Subsequently, the membrane was probed with a polyclonal rabbit antibody against linear MVM-VP epitopes that was diluted 1:2000 in 3 mL TBS-T containing 1 \% milk. The first antibody (reference) was incubated for 1 h at RT. The PVDF membrane was washed in TBS-T for a total 90 min with many buffer replacements. Subsequently, the horseradish peroxidise conjugated secondary antibody (goat $\alpha$-rabbit-HRP, reference) was added for 1 h at RT. This secondary goat anti-rabbit antibody was diluted 1:20000 in TBS-T supplemented with 1 \% milk. To deplete remaining antibodies, the membrane was washed in the same way as described above except for a final wash step with TBS (reference). VP1, VP2, and possibly VP3 were visualized by a chemiluminescence system (SuperSignal West Dura Extended Duration Substrate, reference) following the manufacturer’s instructions. Following this treatment, the PVDF membrane was exposed to a film (Amersham Hyperfilm\textsuperscript{\texttrademark} ECL, reference). Finally, the film was developed using Anatomix Developer Replenisher Solution and Fixer and Replenisher Solution (reference).



\section{Chymotrypsin treatment}

Virus particles were incubated with 0.5 mg/mL chymotrypsin (Sigma) in PBS for 1.5 h at 37 \textcelsius. The reaction was stopped by adding 100 $\mu$M chymostatin (Sigma). Negative controls were incubated in the same buffer for the same time.


\subsection{}
 

%----------------------------------------------------------------------------------------

\section{}


\subsection{}



\subsection{}

