% Chapter 5

\chapter{Genome Architecture} % Main chapter title

\label{Chapter5} % For referencing the chapter elsewhere, use \ref{Chapter2} 

% \lhead{Chapter 2. \emph{Methods}} % This is for the header on each page - perhaps a shortened title

%----------------------------------------------------------------------------------------



\label{sec: Architecture}
The MVM genome is a small, non-permutated, linear, single-stranded DNA molecule \cite{pmid789912, pmid225040, Genome1, Genome2} that is 5085 nt in length for MVMi and 5149 nt for MVMp \cite{pmid3502703}. The relatively long coding sequence of approximately 4.8 kb contains two major, monosense ORFs that span most of the viral genome, with some regions having overlapping coding regions \cite{pmid6298737}. The ORFs encode a non-structural (NS) gene and a structural (VP) gene, by convention termed as occupying the "left" or the "right" half of the coding sequence, respectively. The NS gene encodes four proteins that are required for the replication of the viral genome and are referred to as NS1, NS2\textsuperscript{P}, NS2\textsuperscript{Y}, and NS2\textsuperscript{L}. The VP gene encodes an overlapping set of capsid proteins, VP1 the minor capsid protein and VP2 the major capsid protein \cite{pmid6828378, pmid2939261, pmid2942705}. A representation of the genomic organization of MVM is illustrated in figure~\ref{Architecture}~A, p.~\pageref{Architecture}.   

\section{The MVM Left- and Right-End Telomeres}        

The coding sequence is bracketed by short, imperfect palindromes which form back on themselves to secondary structured duplex telomeres. Both telomeres differ considerably from each other in size, primary sequence and secondary structure \cite{pmid6298737}. Hence, they are physically and functionally disparate and vary in their resolution strategies at the two sites (see section~\ref{Resolution}, p.~\pageref{Resolution}), although the molecular principles that underlie both strategies are very similar \cite{encapsidation}. 

Firstly, the MVM left-end telomere is 121 nt in length and forms into a Y-shaped configuration. The 43 bp stem region only is interrupted by a mismatched bubble sequence where a triplet GAA on the inboard arm is opposed to the dinucleotide sequence GA on the outboard arm. Additionally, an asymmetric thymidine residue is located within the stem on the outboard arm in the immediate proximity to the "ears" that are generated by small internal palindromes. These "ear"-like structures give rise to the Y-shaped configuration of the left-end terminus \cite{pmid225040, pmid6298737, pmid3973977, replication}. A single DNA sequence, designated the "flip" sequence, is conserved in the progeny viral left-end telomere, as is observed \textit{in vivo} \cite{pmid3973977}. The structure of the left-end telomere is depicted in figure~\ref{Architecture}~B (left panel), p.~\pageref{Architecture}. 

Secondly, the MVM right-end telomere is 248 nt in length and is most simply depicted as an almost perfect duplex stem structure of 121 bp. The palindrome only is interrupted by a triplet of unpaired nucleotides that forms a small asymmetric bubble near the distal end of one strand, along with three unpaired bases which form the cross-link at the palindrome axis \cite{pmid6298737, pmid3973977}. As in homotelomeric parvoviruses, two distinct forms of the MVM right-end terminus, referred to as "flip" and "flop", are generated in equimolar amounts \textit{in vivo} (see figure~\ref{Architecture}~D (i) and (ii), p.~\pageref{Architecture}) \cite{telomere2, encapsidation}. These two forms are the inverted complements of one another and both give rise to viral origins, dubbed \textit{oriR} \cite{pmid1388310, pmid9765384, pmid10627544}. A small internal palindrome, surrounding the three-nucleotide bubble, thermodynamically enables an alternative, asymmetric cruciform configuration of the right-end telomere (see figure~\ref{Architecture}~D (iii), p.~\pageref{Architecture}) \cite{pmid6602687}. 


\subsection{Terminal Resolution \textit{versus} Asymmetric Junction Resolution}
\label{Resolution}

As is the case for most of the heterotelomeric parvoviruses, MVM shows packaging bias with minus strands preferentially encapsidated to plus strands by a 10-100-fold margin \cite{pmid6828378, pmid3296697}. This results from differences in the efficiency of their two DNA replication origins at both ends of their genomes, rather than any strand-specific packaging sequence. In particular, the efficient nick site of the \textit{oriR} dictates the negative polarity of the packaged strand which is encapsidated in MVM virions \cite{pmid15681430}. 

Given the fact that alike their homotelomeric cousins, the right-end hairpin of MVM exists as an equimolar mix of flip and flop sequence orientations, it is processed by a similar terminal resolution strategy. Nicking of the hairpin near the junction between palindromic and non-palindromic sequences and subsequent extension of the right-end terminus allows an efficient inversion of the palindrome (see step iii and iv in figure~\ref{RHR}, p.~\pageref{RHR}) On the contrary, the MVM left-end hairpin predominates in the flip orientation, indicating its generation by an asymmetric junction resolution mechanism \cite{pmid12743281}. Briefly, the asymmetric bubble sequence in the stem of the MVM left-end telomere prevents assembly of an active nicking complex. Thus, the left-end telomere can not function as a replication origin in its hairpin conformation \cite{pmid8995615}. During rolling hairpin replication (see section~\ref{Replication}, p.~\pageref{Replication}), the hairpin is unfolded, extended, and copied to form the fully basepaired, imperfect palindromic junction sequence which bridges adjacent genomes in an intermediate dimer RF (see figure~\ref{Architecture}~B (right panel), p.~\pageref{Architecture}). It was demonstrated that such junctions can initiate DNA replication in a NS1-dependent manner \cite{pmid8076610, pmid1530771}. Formation of the dimer junction effectively segregates two potential origins of DNA replication, one derived from each arm of the hairpin, on either side of the junction's symmetry axis. However, only one of these is active. The activity is regulated by the sequence of the asymmetric bubble which serves as a precise spacer between the NS1 binding site and the parvovirus initiation factor (PIF). Binding of which stabilizes the interaction of NS1 with the active (TC) origin (\textit{OriL\textsubscript{TC}}) but not with the inactive (GAA) origin (\textit{OriL\textsubscript{GAA}}) \cite{pmid11435581}. The minimal left-end origin of replication, dubbed \textit{oriL}, is illustrated in figure~\ref{Architecture}~C, p.~\pageref{Architecture}. It extends from two 5'-ACGT-3' motifs which represent binding sites for PIF \cite{pmid8995666, pmid9223459, pmid10523663}, to a 5'-(ACCA)\textsubscript{2}-3' binding site for the viral initiator nickase, NS1 \cite{pmid7853501}, to the active nick site \cite{pmid8076610}. Recent studies revealed that MVM tolerates both sequence and orientation changes in its left-end hairpin. From this follows that maintaining the flip orientation of the left-end telomere is a consequence of, but not the reason for, asymmetric dimer junction resolution. However, the same study indicated that asymmetric left-end processing is crucial for MVM replication \cite{pmid22933276}.  

In summary, the heterotelomeric hairpins, together with a few adjacent nucleotides, provide all of the \textit{cis}-acting information required for both efficient genome replication and encapsidation. In particular, these terminal nucleotides, representing less than 10~\% of the entire genome, create the replication origins by providing nicking sites that will allow the DNA to be nicked and used as a primer or effectively separate unit-length genomes for DNA packaging. Additionally, they function as flexible hinge regions that are used to establish and re-orient the replication fork, allowing it to roll back and forth along the linear viral DNA \cite{telomere2, telomere3, handbook, RHR}.        

\section{Genetic variability}

When compared with cellular DNA, the genome of MVM has a relatively high GC-content (42~\%), partially reflecting its high density of regulatory elements. The complexity of the viral genome is increased by transcriptional promoter sequences and various splicing signals that lie embedded within the same primary sequence, beyond the encoded proteins which are organized in multiple overlapping ORFs. Nevertheless, following inoculation of clonal populations of MVMi stocks in mice, genetically disparate antibody-escape variants emerged \textit{in vivo}. This indicates that viral replication appears to support the generation of heterogeneity \cite{pmid12552010}. Another example concerns the emergent branch of CPV during its evolution from FPV since 1978. The substitution rate of CPV resembles that seen in rapidly evolving RNA viruses, as for example HIV-1 and human influenza A virus \cite{pmid15626758}. Remarkably, such diversity occurred despite the fact that the viral genome is multiplied by a subset of the host's DNA replication machinery, hence the mutation rates would be expected to be low. Probably, the unidirectional strand-displacement mechanism may exhibit lower fidelity compared to the bidirectional replication of eukaryotic genes. Additionally, the concatemeric duplex intermediates may allow for inter- and intramolecular recombination during replication of the viral DNA. Moreover, there are several lines of evidence that MVM exploits the DNA damage response (DDR) machinery early in infection in order to enhance its replication and to ameliorate virus-induced cell cycle arrest in the S-phase \cite{pmid20949077}. Therefore, it seems possible that under such conditions the replication forks appear error-prone. Finally, environmentally induced changes in the viral DNA sequence, such as depurination or deamination, can not be corrected because virions contain ssDNA and hence do not provide a template for excision or mismatch repair systems. Nonetheless, the genetic complexity, in consequence of the constrained genome size, severely and selectively restricts the types of tolerated modifications \cite{telomere}.   


\begin{figure}
\centering
  \includegraphics[scale=0.7]{Architecture}
  \vspace{-8em}
  \caption[Genome architecture of minute virus of mice (MVM).]
   {Genome architecture of MVM. \textbf{(A)} The terminal hairpins, drawn to represent their predicted structures, are scaled approximately 20x relative to the rest of the genome. Major open reading frames are represented by arrowed boxes and alternative RNA splicing for NS2 is indicated. Proteins are shaded green for the major replication initiator protein (NS1), blue for the structural (VP) proteins of the capsid, and yellow for sequences unique to the ancillary non-structural proteins. The two transcriptional promoters, P4 and P38, are indicated by rightward arrows and the polyadenylation site by the AAAAA-sequence block. Abbreviations: SAT, small alternatively translated protein \cite{small}. \textbf{(B)} The left-end hairpin of MVM and the dimer junction are shown in diagrammatic form. Asymmetries such as the "ear"-like structures, extra-helical T, and bubble sequence are indicated. The fully duplex, dimer junction, generated by rolling hairpin replication (see section~\ref{Replication}, p.~\pageref{Replication}), is shown on the right hand side. The short, palindromic sequences derived from the hairpin ears are represented by cross-hatched boxes. The active \textit{OriL\textsubscript{TC}} is boxed, with an arrow indicating the nick site. The equivalent sequence generated on the GAA side of the bubble is framed by a dashed box with an arrow at the potential nick site that is crossed out to indicate that \textit{OriL\textsubscript{GAA}} is not active \cite{pmid12885883, pmid16928767}. \textbf{(C)} Sequence details of the active left-end origin (approx. 50 bp) are shown, with an arrow indicating the active nick site. The minimal sequence required for origin activity is indicated by the double-headed arrow. Sequences of the bubble and the PIF, CRE, and NS1 binding sites are indicated. An asterisk represents the position of the extra-helical T, now base paired, and the gray box below it indicates the CRE consensus sequence \cite{pmid12885883}. \textbf{(D)} Alternate conformations of the right-end hairpin sequences of MVM. The right-end terminus can form a hairpin structure in either the flip (i) of flop (ii) sequence orientation or a cruciform structure (iii). In the cruciform configuration, the binding sites for the replicator protein, NS1, are boxed and their site of nucleolytic cleavage is represented by a vertical arrow \cite{pmid8614999}.
} 
\label{Architecture}
\end{figure}


\nomenclature{VP1}{Viral protein 1}
\nomenclature{VP2}{Viral protein 2}
\nomenclature{VP3}{Viral protein 3}
\nomenclature{NS}{Non-structural (protein)}
\nomenclature{PIF}{Parvovirus initiation factor}
\nomenclature{CRE}{cAMP-responsive element}
\nomenclature{SAT}{Small alternatively translated protein}
\nomenclature{DDR}{DNA damage response}